src/util/simplify_expr_pointer.cpp

```diff
@@ -9,6 +9,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "simplify_expr_class.h"
 
 #include <cassert>
+#include <goto-symex/goto_symex.h>
```

Comment:
I suspect that's a bad cross-module dependency (but then, so is utils knowing about symex internal magic)

Reply:
Yeah looks like this particular simplification shouldn't be in util. Happy to either remove this particular usage - or pull the macro into `util`?